shim.js

```diff
@@ -21,7 +19,7 @@ module.exports = function shimName() {
 			configurable: true,
 			enumerable: false,
 			get: function () {
-				var name = fnCall(polyfill, this);
+				var name = polyfill.bind(this).call(this);
```

Comment:
This relies on a native `.bind` implementation, as well as an unmodified `.bind` and `.call`; I'd like to find a way to make it work with `function-bind`.